docker-compose.yml

```diff
@@ -3,8 +3,11 @@ services:
     container_name: inatator_db
     image: postgres:16
     restart: always
-    env_file:
-      - .docker.env
+    environment:
+      - POSTGRES_DB
```

Comment:
after changing the `env_file` to these ` environment` variables, do we need to run app and docker compose locally differently?
in other words, do we need to change the instructions in the README file?

Reply:
Yes, nice catch, the Readme instructions should be updated as well!